docker/README.md

```diff
@@ -0,0 +1,30 @@
+# Docker image for sendgrid-ruby
+
+## Quickstart
+1. [Install Docker](https://docs.docker.com/engine/installation/) on your machine.
+2. Run `docker run --rm -it sendgrid/sendgrid-ruby irb`.
```

Comment:
What is the `irb` at the end there?

Reply:
The idea was to get a interpretor where a dev could try out the gem without having to run it.